README.md

```diff
@@ -31,6 +31,21 @@ services.AddEventuateTramSqlProducer(dbSchemaName, (provider, o) =>
 	});
 ```
 
+### Configuration Properties
+- Customize the maximum amount of time (ms) to poll for a new event (default 100):
+```c#
+PollTimeout = 500; // default: 100
+```
+- Customize the Backpressure behavior. Message consumption will be paused once the size of the unprocessed message
+- queue exceeds the PauseThreshold and resumed once it drops back below the ResumeThreshold.
+```c#
+BackPressure = new BackPressureConfig
+{
+    PauseThreshold = 100, // default: int.MaxValue
```

Comment:
If you choose to change to uint.MaxValue, update here as well.

Reply:
done